lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -132,8 +134,10 @@ serveWallet (cfg, tr) databaseDir hostPref listen lj beforeMainLoop = do
             waitForService "Jörmungandr" tr nPort $
                 waitForNetwork nl defaultRetryPolicy
             let (_, bp) = staticBlockchainParameters nl
-            rndApi <- apiLayer tr (toWLBlock <$> nl)
-            seqApi <- apiLayer tr (toWLBlock <$> nl)
+            let rndTl = newTransactionLayer @n (getGenesisBlockHash bp)
+            let seqTl = newTransactionLayer @n (getGenesisBlockHash bp)
```

Comment:
~Why do the transaction layers need to be different? `newTransactionLayer` is passed the same arguments.~

Ah, right! The implicit key type-param!